src/openms/include/OpenMS/DATASTRUCTURES/LPWrapper.h

```diff
@@ -421,6 +416,11 @@ namespace OpenMS
       @param[in] row_index Index of the row
       @param[in] column_index Index of the column
       @param[in] value Value to set at the specified position
+      
+      @throws Exception::NotImplemented for HIGHS solver (sparse matrix modification not supported)
```

Comment:
See my note in the PR. Technically HIGHS would support setElement for all non-zero elements but I haven't implemented that here. None of our code uses setElement anyhow.